simulator/infra/instrcache/LRUCache.h

```diff
@@ -13,16 +13,16 @@
 #include <utility>
 
 #include <infra/types.h>
+#include <infra/lru/LRUCacheInfo.h>
```

Comment:
My proposal is to split this commit into 2:
1. Movement of LRUCacheInfo to a new file, CMake alignments etc. — should pass without any problem
2. Integration of LRUCacheInfo to LRUCache

Reply:
done